src/docs/user/references/Notification-service.adoc

```diff
@@ -353,22 +353,16 @@ INFO, WARNING, ERROR, CRITICAL. The default severity is INFO.
 [cols="2,2,2,5,5"]
 |===
 | *Type* | *Readable name* | *Severity* | *Message* | *Description*
-| `ASSOCIATION_FAILED`
-| Association failed
+.3+| `ASSOCIATION_FAILED`
+.3+| Association failed
```

Comment:
what is .3+ ? and why we have .2+ later?

Reply:
This is to merge cells in tables like this

![image](https://user-images.githubusercontent.com/53172400/120236065-b934f480-c25b-11eb-8db6-92486d4a0155.png)